src/gfx_layout.cpp

```diff
@@ -596,10 +596,8 @@ static inline void GetLayouter(Layouter::LineCacheItem &line, const char *&str,
 			state.SetColour((TextColour)(c - SCC_BLUE));
 		} else if (c == SCC_PREVIOUS_COLOUR) { // Revert to the previous colour.
 			state.SetPreviousColour();
-		} else if (c == SCC_TINYFONT) {
-			state.SetFontSize(FS_SMALL);
-		} else if (c == SCC_BIGFONT) {
-			state.SetFontSize(FS_LARGE);
+		} else if (c >= SCC_NORMALFONT && c <= SCC_MONOFONT) {
```

Comment:
Personally I'd prefer to keep individual if statements rather than checking for a range, that way it doesn't matter if codes get rearranged.

Reply:
Individual statements get out of hand; 2 is okay, 4 is a crowd. Possibly delegate it to a subfunction (via a switch or what-ever)?

Reply:
True. Maybe the keep as it is and add a comment to the control code list.

Reply:
Owh, 4 lines higher there is already a range :D Seems that ship already has sailed :)